exporter/awscloudwatchlogsexporter/exporter.go

```diff
@@ -252,3 +268,35 @@ func attrValue(value pcommon.Value) interface{} {
 		return nil
 	}
 }
+
+func (e *exporter) getPusher(logGroup, logStream string) cwlogs.Pusher {
+	e.pusherMapLock.Lock()
+	defer e.pusherMapLock.Unlock()
+
+	var ok bool
+	var streamToPusherMap map[string]cwlogs.Pusher
+	if streamToPusherMap, ok = e.groupStreamToPusherMap[logGroup]; !ok {
```

Comment:
```suggestion
        streams, ok := e.groupStreamToPusherMap[logGroup]
        if !ok {
```
Simplifies variable creation.

Reply:
Once again, a copy paste job from EMF.  It someone mirroring all these changes over there?

Reply:
also nobody noticed the redunant emf pusher var haha